### PR TITLE
Make language actually working

### PR DESCRIPTION
**Pull request by Blueberryy**
+extra Russian translation :p

### CODE REVIEW

File: resource/localization/en/adv_dublicator_tool.properties [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
```
Comment (TomyLobo) [inline resource/localization/en/adv_dublicator_tool.properties:1]: the file name is misspelled, please correct

File: resource/localization/ru/adv_dublicator_tool.properties [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
```
Comment (TomyLobo) [inline resource/localization/ru/adv_dublicator_tool.properties:1]: the file name is misspelled, please correct

**Review by TomyLobo — CHANGES_REQUESTED** [review]

**Comment by Blueberryy** [comment]
ah sorry, will fix soon

**Comment by Blueberryy** [comment]
fixed

**Comment by thegrb93** [comment]
I think we'll need a resource.AddWorkshop for this now so the new files get acquired.

**Comment by Blueberryy** [comment]
Files downloading if you subbed to addon, i don't think we actually need to add this

**Comment by thegrb93** [comment]
What if we instead do
```lua
if GetConvarString("whatever_language_convar_is")=="ru" then
    language.Add("...","russian")
else
    language.Add("...","english")
end
```

**Comment by thegrb93** [comment]
It does

**Review by thegrb93 — CHANGES_REQUESTED** [review]
Use `GetConvarString("gmod_language")` instead of .properties files.

**Pull request closed without merging**